### PR TITLE
Update the preview action for pull_request.yml

### PR DESCRIPTION
**Pull request by cansavvy**
# Description

 

There was some stuff on OTTR website that didn't make it here. 

 

Propagating changes from https://github.com/jhudsl/OTTR_Template_Website since metricminer-dashboard is a flavor of an OTTR Template Website.

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -169,3 +172,8 @@ jobs:
 
             _Updated at ${{ steps.build-components.outputs.time }} with changes from ${{ steps.build-components.outputs.commit_id }}_
           edit-mode: replace
+          body: |
+            The latest commit did not produce rendering changes.
+
+            _Updated at ${{ steps.build-components.outputs.time }} with changes from ${{ steps.build-components.outputs.commit_id }}_
+          edit-mode: replace
```
Comment (kweav) [inline .github/workflows/pull_request.yml]: Is this section just a copy of what's above in lines 170-174?

Comment (cansavvy) [reply .github/workflows/pull_request.yml]: Oh yeah not sure why that happened. Good catch.

**Review by kweav — APPROVED** [review]
Think it should be good -- excited that it adds in the ability to download the preview!

**Comment by kweav** [comment]
@cansavvy good to merge?

**Comment by cansavvy** [comment]
Thanks for bringing this to my attention @kweav !

**Pull request merged**